hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/AllocationFileLoaderService.java

```diff
@@ -605,13 +606,15 @@ private void loadQueue(String parentName, Element element,
 
     queueAcls.put(queueName, acls);
     resAcls.put(queueName, racls);
-    if (maxQueueResources.containsKey(queueName) &&
-        minQueueResources.containsKey(queueName)
-        && !Resources.fitsIn(minQueueResources.get(queueName),
-            maxQueueResources.get(queueName))) {
+    ResourceConfiguration maxResourceConf = maxQueueResources.get(queueName);
+    if (maxResourceConf != null &&
+        !maxResourceConf.isPercentage() &&
```

Comment:
A warning is necessary even when the config is percentage-based. I would argue the warning is more important in this case than the other.